mm2src/mm2_main/src/mm2.rs

```diff
@@ -38,6 +38,7 @@
 #[macro_use] extern crate serde_json;
 #[macro_use] extern crate serde_derive;
 #[macro_use] extern crate ser_error_derive;
+extern crate core;
```

Comment:
Why?

Reply:
Have no idea how this was added. Will remove it.